org.asamk.SignalCli.json

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "org.asamk.SignalCli",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk21" ],
+  "command": "signal-cli",
+  "finish-args": [
+    "--own-name=org.asamk.Signal",
+    "--talk-name=org.asamk.Signal",
```

Comment:
This should be fixed upstream... The id and bus names you use internally should match with the Flatpak id too.

Reply:
I've removed them.